### PR TITLE
add config.RequiresBackports

### PR DESCRIPTION
**Pull request by tych0**
This will be used for the implementation of
https://github.com/juju/juju/pull/4191 the LXD container type.

Signed-off-by: Tycho Andersen <tycho.andersen@canonical.com>

### CODE REVIEW

File: packaging/config/functions.go [hunk 1]
```diff
@@ -27,3 +27,18 @@ func GetCloudArchiveSource(series string) (packaging.PackageSource, packaging.Pa
 		return configureCloudArchiveSourceUbuntu(series)
 	}
 }
+
+func RequiresBackports(series string, pkg string) bool {
+	backportPkgs, ok := backportsBySeries[series]
```
Comment (axw) [inline packaging/config/functions.go]: This `ok` check is unnecessary, you'll just range over a nil slice below and then return false anyway.

**Comment by axw** [comment]
LGTM with one suggestion.

**Comment by tych0** [comment]
On Tue, Jan 26, 2016 at 03:46:17PM -0800, Andrew Wilkins wrote:
> LGTM with one suggestion.

Thanks, I've pushed a fix.

**Comment by axw** [comment]
$$merge$$

**Pull request merged**